uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -478,7 +486,7 @@ Arguments:
 sub runCommand {
     my $this = shift;
     my ($command) = @_;
-    print "\n\n $command \n\n " if $this->{'verbose'};
+    print "\n\n $command \n\n " if $this->{verbose};
```

Comment:
Okay, so line 465 above ( line 457 in the old file) should be changed then to $this->{verbose} as well.